xbmc/addons/gui/skin/SkinTimerManager.cpp

```diff
@@ -20,19 +20,6 @@
 
 using namespace std::chrono_literals;
 
-CSkinTimerManager::CSkinTimerManager() : CThread("SkinTimers")
-{
-}
-
-void CSkinTimerManager::Start()
-{
-  std::unique_lock<CCriticalSection> lock(m_timerCriticalSection);
```

Comment:
Not sure we still need `m_timerCriticalSection` as we are not calling `CSkinTimermanager`from other threads than the main thread.

Reply:
shouldn't be needed anymore, it's all called from the main thread now. Added a note to doxy docs warning about this.